libraries/hermes/src/Rest/SwaggerParser.ts

```diff
@@ -140,13 +136,13 @@ export class SwaggerParser extends ConduitParser<ParseResult, ProcessingObject>
     description?: string,
   ): void {
     if (description && name === 'body') {
-      // @ts-ignore
+      processingObject.properties = processingObject.properties || {};
```

Comment:
remove the added lines "processingObject.properties = processingObject.properties || {};" after making properties not optional. The "getProcessingObject" function always returns a properties object in there.